### DISCUSSION TITLE
Live Voting in Openslides 3.4

### DISCUSSION THREAD

**Discussion by AProbst82** [OpenSlides/OpenSlides#5991] [Q&A] [answered]
Hello, I hope this is the right place to ask this question:
I have installed OpenSlides 3.4 on a Docker container, but when I try to create an election, I can only choose "analog"; Live voting is not possible. Am I missing a Plug-in or similar?

Also, if there is a manual beyond the wiki pages in this project, I'd love a link

Thanks for your hard work - this seems like it will make our general assembly a lot easier.

**Comment by ManniLoe** [comment, marked as answer]
Hi, the electornic voting not enabled by default.
Take a look at the .env file or export ENABLE_ELECTRONIC_VOTING=True bevore the m4 command. Then the electronic voting is enabled and you can select more then "analog"

**Reply by AProbst82** [reply]
Thank you, that was the solution! I'll give feedback once I have survived the general assembly :D
